source/Cosmos.Core/Memory/RAT.cs

```diff
@@ -297,6 +297,11 @@ public static uint GetFirstRATIndex(void* aPtr)
             throw new Exception("Page type not found. Likely RAT is rotten.");
         }
 
+        /// <summary>
+        /// Get the pointer pointing to the start of the position to which the current pointer is pointing

```

Comment:
if im understanding this right, would this not be better "get the pointer of the start of the page of this pointer"

Reply:
Updated it